assets/css/styleguide.scss

```diff
@@ -308,8 +304,15 @@ a#skipnav {
   }
 
   p, a {
```

Comment:
@msecret this was one of those "styleguide-only" styles and was done to accomplish this trello card: https://trello.com/c/cYuS0RJm/447-clean-up-typography-on-mobile-footer

It was added to style our super bare footer, which is only text. Are you saying add a note to switch to use classes instead?